### PR TITLE
doc: update readme

### PR DESCRIPTION
**Pull request by samen93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -457,6 +457,10 @@ The following attributes are exported in the outputs of the module:
 
   All attributes of the created `google_storage_bucket` resource.
 
+- **`iam`**
+
+  The `iam` resource objects that define the access to the resources.
```
Comment (katcipis) [inline README.md:462]: Given that the module is specific to GCS:

```suggestion
  The `iam` resource objects that define the access to the GCS bucket.
```

File: outputs.tf [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 # remap iam to reduce one level of access (iam[]. instead of iam[].iam.)
 output "iam" {
-  description = "The iam resource objects that define the access to the secret"
+  description = "The iam resource objects that define the access to the resources."
```
Comment (katcipis) [inline outputs.tf:7]:
```suggestion
  description = "The iam resource objects that define access to the GCS bucket."
```

**Review by katcipis — COMMENTED** [review]
Maybe the doc could be more precise on which resource specifically the IAM is giving access to.

**Review by mariux — APPROVED** [review]
lgtm

**Review by katcipis — APPROVED** [review]

**Pull request merged**